packages/@react-native-windows/tester/src/js/examples-win/Accessibility/AccessibilityInfoExample.js

```diff
@@ -68,6 +69,45 @@ function AccessibilityInfoExample(props): React.Node {
   );
 }
 
+function TouchableExample(props): React.Node {
+
+  let myElement = React.createRef();
+
+  return (
+    <View>
+      <TouchableWithoutFeedback
+        onPress={() => {
+          const reactTag = findNodeHandle(myElement.current);
+          if (reactTag) {
+            AccessibilityInfo.setAccessibilityFocus(reactTag);
+          }
+        }}
+        accessibilityLabel="TEST Set Accessibility Focus"
+        {...{
+          focusable: true,
+        }}>
+        <View>
+          <Text>TEST setAccessibilityFocus</Text>
```

Comment:
Nit; Just checking if `TEST` is intentionally all-caps...

Reply:
good call, not entirely intentional 😅